### DISCUSSION TITLE
Motion Detection without Objects

### DISCUSSION THREAD

**Discussion by wetty** [blakeblackshear/frigate#1871] [General]
Hello,

I have build a birdbox/nesting box with integrated camera. Yesterday a sparrow visited the box, but frigate did not detect the bird. 
So my question is whether I can disable object detection to only detect motion?

regards 
Thomas

**Comment by j0rd** [comment]
I'd also like motion detection as an event with out objects.

Objects works well for me, but sometimes you just want motion as an event.

**Comment by curif** [comment]
I don't have a Coral, and it's impossible to get one in my country. Then, I fallback on CPU to get object detection working. But I wan't to disable it too (it's almost imposible to use in CPU mode) and I don't know how to do it in the 0.9+. Some one with an example? 

Thank you.

**Reply by bacon-avenger** [reply]
> I don't have a Coral, and it's impossible to get one in my country. Then, I fallback on CPU to get object detection working. But I wan't to disable it too (it's almost imposible to use in CPU mode) and I don't know how to do it in the 0.9+. Some one with an example?

What do you have for a CPU?  It might not be strong enough.

I'm running Frigate as installed via the community addon in home assistant under ESXi 7.x at the moment with two threads assigned to the ubuntu VM everything runs in.  Object detection isn't the fastest, and sometimes I think it misses things, but it does work fairly well.  The relevant lines in the config are:

```
detectors:
  cpu1:
    type: cpu
  cpu2:
    type: cpu
```
CPU's I have in the host are old E5-2660 Xeon's (think Sandy Bridge era)

**Reply by upsangel** [reply]
I have the same thought, I am running Frigate on ARM board the cpu is very weak. No idea how to set Frigate as a non-AI detection NVR.

**Comment by madmic1314** [comment]
+1 Be great to have this - would mean scrolling through recordings you could look for motion events only.

**Comment by madmic1314** [comment]
Been giving this some thought whilst playing with integrating to deepstack. My idea is to call HA image snapshot with a timestamped filename everything there is motion and Frigate is processing an image and save this into the HA media folder. Will need another process to clear this down, but one thing at a time!

Hopefully @blakeblackshear will add in an option to tag motion as an event, even if nothing was detected. I might open a request, rather than leave in a chat?

**Reply by Sam-Whitaker** [reply]
Hi @madmic1314, Did you submit this as a request?
I'm also pretty interested in having motion as an event. I'd like to know if there's been motion of any kind and it seems like it'd be pretty useful to have that as an event to trigger recording.

**Reply by madmic1314** [reply]
No never did - might put it in, don't think there is one for this. Worth putting in, will do late on

**Comment by madmic1314** [comment]
https://github.com/blakeblackshear/frigate/issues/2231 added

**Reply by Sam-Whitaker** [reply]
Great job! Thank you :D
